### PR TITLE
added unittests for delete-command

### PR DESCRIPTION
**Pull request by heloszia**
this is my first contribution to a project on github 😄

### CODE REVIEW

File: src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace UnitTests.Core.Commands.DeleteCommandTests
+{
+
+    public class DeleteCommandShould
+    {
+        private readonly Mock<IRepository> _RepositoryMock = new Mock<IRepository>();
```
Comment (benrick) [inline src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs]: rename to `_repositoryMock` and adjust others as well

File: src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+using DevChatter.Bot.Core.Commands.Operations;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Data.Specifications;
+using DevChatter.Bot.Core.Events.Args;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace UnitTests.Core.Commands.DeleteCommandTests
```
Comment (benrick) [inline src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs]: Move this class into the `Operations.DeleteCommandOperationTests` folder and namespace

File: src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+{
+
+    public class DeleteCommandShould
+    {
+        private readonly Mock<IRepository> _RepositoryMock = new Mock<IRepository>();
+        private readonly IList<IBotCommand> _AllCommands = new List<IBotCommand>();
+        private readonly SimpleCommand _QuoteCommand;
+        private readonly CommandReceivedEventArgs _CommandReceivedEventArgs;
+
+        public DeleteCommandShould()
```
Comment (benrick) [inline src/UnitTests/Core/Commands/DeleteCommandTests/DeleteCommandShould.cs]: `TryToExecuteShould`

File: src/UnitTests/UnitTests.csproj [hunk 4]
```diff
@@ -20,4 +20,8 @@
     <ProjectReference Include="..\DevChatter.Bot.Infra.Twitch\DevChatter.Bot.Infra.Twitch.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (benrick) [inline src/UnitTests/UnitTests.csproj]: What is this change? Is it needed?

**Comment by benrick** [comment]
It would be good to include in this the other path as well, which is that when the non-mod does this, it doesn't have these effects.

**Comment by heloszia** [comment]
I fixed the issues and added two more tests

**Pull request merged**